src/TestSessionEnvironment.php

```diff
@@ -618,4 +622,17 @@ public function waitForPendingRequests($await = 700, $timeout = 10000)
 
         return true;
     }
+
+    private function connectToDB(array $databaseConfig): void
+    {
+        if (method_exists(DB::class, 'hasConfig')) {
+            // CMS 5.4+ - ensure we connect the primary connection and not a replica
+            // which can happen if we use the default value of DB::CONN_DYNAMIC
+            // and there is a replica database configured
+            DB::connect($databaseConfig, DB::CONN_PRIMARY);
```

Comment:
This PR targets CMS 6 now. We don't need this condition (and therefore probably don't need this method at all)

Reply:
Updated